src/configure/configurers/azurePipelineConfigurer.ts

```diff
@@ -42,16 +42,6 @@ export class AzurePipelineConfigurer implements Configurer {
 
     public async getInputs(inputs: WizardInputs): Promise<void> {
         try {
-            if (inputs.sourceRepository.repositoryProvider === RepositoryProvider.Github) {
```

Comment:
we should not make any change in AzurePipelines creation flow and this guy will not need PAT for repo analysis.